src/dynatrace/opentelemetry/metrics/export/serializer.py

```diff
@@ -271,6 +279,25 @@ def _remove_control_characters(cls, s: str) -> str:
     @classmethod
     def _normalize_dimension_value(cls, value: str):
         value = value[:cls.__dv_max_length]
-        value = DynatraceMetricsSerializer._remove_control_characters(value)
+        value = cls._remove_control_characters(value)
         value = cls.__re_dv_escape_chars.sub(r"\\\g<1>", value)
         return value
+
+    @classmethod
+    def _make_unique_dimensions(cls, labels: Iterable[Tuple[str, str]],
+                                dimensions: Iterable[Tuple[str, str]]):
```

Comment:
Please document the precedence applied here.

Reply:
should be done now.